tools/automation.py

```diff
@@ -229,20 +229,64 @@ def xlsx_to_yaml_parser(mentor_row, mentor_index):
     return mentor
 
 
-def get_all_mentors_in_yml_format(xlsx_file_path, skip_rows=0):
+def get_yml_data(yml_file_path):
```

Comment:
This is a new method.  It reads yml file and return dataftame with mentor name, index, sort and disabled values.